### PR TITLE
Generate execution result as part of block computation

### PR DESCRIPTION
**Pull request by pattyshack**
Also removed handleComputationResult since it does nothing

### CODE REVIEW

File: engine/execution/computation/computer/result_collector.go [hunk 1]
```diff
@@ -259,5 +262,11 @@ func (collector *resultCollector) Finalize() (
 		return nil, err
 	}
 
+	collector.result.ExecutionResult = flow.NewExecutionResult(
+		collector.parentBlockExecutionResultID,
+		collector.result.ExecutableBlock.ID(),
+		collector.result.Chunks,
+		collector.result.ConvertedServiceEvents,
+		collector.result.ExecutionDataID)
```
Comment (pattyshack) [inline engine/execution/computation/computer/result_collector.go:270]: note that this id is currently incorrectly.  The id is correctly populated in #3927

File: engine/execution/ingestion/engine.go [hunk 2]
```diff
@@ -611,9 +614,23 @@ func (e *Engine) executeBlock(ctx context.Context, executableBlock *entity.Execu
 	span, ctx := e.tracer.StartSpanFromContext(ctx, trace.EXEExecuteBlock)
 	defer span.End()
 
+	parentID := executableBlock.Block.Header.ParentID
+	parentErID, err := e.execState.GetExecutionResultID(ctx, parentID)
+	if err != nil {
+		lg.Err(err).Msg(
```
Comment (janezpodhostnik) [inline engine/execution/ingestion/engine.go:620]: use `Msgf` instead or better yet 
`.Str("parentID", parentID)`

**Review by janezpodhostnik — APPROVED** [review]

**Review by zhangchiqing — APPROVED** [review]

**Comment by pattyshack** [comment]
bors merge

**Comment by pattyshack** [comment]
bors merge

**Pull request merged**